data/templates/article-wrapper.mst

```diff
@@ -15,6 +15,7 @@
         <script type="text/javascript" src="resource:///com/endlessm/knowledge/data/templates/js/{{{.}}}"></script>
         {{/javascript-files}}
 
+        <script type="application/json" id="content-metadata">{{{content-metadata}}}</script>
```

Comment:
@ptomato you mean something like

```
<script type="text/javascript">
window.contentMetadata = JSON.parse({{{content-metadata}}});
</script>
```

So they just need to window.contentMetadata ? (and not having to JSON.parse each time?)